### PR TITLE
Converting nVFats to nVFATS everywhere

### PR DESCRIPTION
**Pull request by mexanick**
Providing a name standartization/typo fixes. The code has been tested in venv on QC7 PC. 
### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: utils/scurveAlgos.py [hunk 1]
```diff
@@ -306,15 +306,15 @@ def anaUltraScurve(args, scurveFilename, calFile=None, GEBtype="short", outputDi
...
     # Create the fitter
     from gempython.gemplotting.fitting.fitScanData import ScanDataFitter
     if performFit:
         fitter = ScanDataFitter(
-                calDAC2Q_m=calDAC2Q_Slope, 
+                calDAC2Q_m=calDAC2Q_Slope,
                 calDAC2Q_b=calDAC2Q_Intercept,
                 isVFAT3=isVFAT3,
-                nVFats = nVFATS,
+                nVFATS = nVFATS,
```
Comment (jsturdy) [inline utils/scurveAlgos.py]: clean up extra whitespace conflicting with nearby style?

**Review by jsturdy — CHANGES_REQUESTED** [review]
minor cleanup, probably rebase on top of #283 once its merged

**Comment by jsturdy** [comment]
ready for rebase following fixup

**Comment by mexanick** [comment]
should be ready to go

**Review by jsturdy — APPROVED** [review]
:+1: (also for the extra whitespace cleanup, which i ignored for the review :-D)

**Pull request merged**